packages/plugin-ext/src/main/node/plugin-cli-contribution.ts

```diff
@@ -23,8 +23,13 @@ import { LocalDirectoryPluginDeployerResolver } from './resolvers/plugin-local-d
 export class PluginCliContribution implements CliContribution {
 
     static PLUGINS = 'plugins';
+    static MAX_SESSION_LOGS_FOLDERS = 'max-session-logs-folders';
```

Comment:
please prefix with plugin, Theia has primary logs and this does affect them

Reply:
done